packages/fxa-settings/src/components/Settings/ProductPromo/en.ftl

```diff
@@ -3,5 +3,7 @@
 product-promo-monitor =
  .alt = { -product-mozilla-monitor }
 product-promo-monitor-description = Find where your private info is exposed — and take it back
+product-promo-monitor-plus-description = Privacy Matters: Find where your private info is exposed and take it back
 # Links out to the Monitor site
 product-promo-monitor-cta = Get free scan
+product-promo-monitor-plus-cta = Get started
```

Comment:
Can you add a comment with context for this? "Get started" could be translated differently, so info on what clicking the CTA would be "starting" would help.